Collections-Endpoint.md

```diff
@@ -21,7 +21,9 @@ Item properties :
 - `title` is a single string.   Additional descriptions may be placed in `dts:dublincore` using `dc:title`, e.g. for internationalization.
 - `@id` is the identifier of the object (TODO: add language recommending the use of URIs for ids)
 - `@type` is either `Collection` or `Resource`
-- `totalItems` is the number of children contained by the object
+- `totalItems` - total number of items that you can find in the members property
```

Comment:
could we clarify that this is across all pages? e.g. "total number of items that you can find in the members property across all pages" or "total number of items that you can find in the members property irrespective of pagination" ?

Reply:
sure !